linkml_runtime/utils/schemaview.py

```diff
@@ -138,9 +138,7 @@ def is_absolute_path(path: str) -> bool:
     :return: true or false
     :rtype: bool
     """
-    if path.startswith("/"):
-        return True
-    if "://" in path:
+    if path.startswith("/") or "://" in path:
```

Comment:
whichever idiot added this `"://" in path` expression didn't notice they could have combined it with the expression above